src/main/java/io/meeds/qa/ui/pages/page/factory/application/ApplicationPage.java

```diff
@@ -88,7 +94,7 @@ public boolean isApplicationVisible(String application) {
   public static BaseElementFacade elementChallengeApplicationPage;
 
   private BaseElementFacade addApplicationAFavoriteInApplicationCenter(String app) {
-    return findByXpath(String.format("(//*[@class='authorisedAppContent']//*[@class='appTitle']//div[contains(text(),'%s')]/following::*[contains(@class, 'applicationActions')]//i[contains(@class, 'mdi-star')])",
+    return findByXpath(String.format("(//*[@class='authorisedAppContent']//*[@class='appTitle noHelpPage' and contains(text(),'%s')]/following::*[contains(@class, 'applicationActions')]//i[contains(@class, 'mdi-star')])[01]",
```

Comment:
use `contains(class, 'appTitle')` instead of equality operator `@class='appTitle noHelpPage'` to make this test more pertinent

Reply:
done